features/webxr-layers.yml

```diff
@@ -0,0 +1,71 @@
+name: WebXR Layers
+description: To be written...
+spec: https://immersive-web.github.io/layers/
+group: webxr
+status:
+  compute_from: api.XRCompositionLayer
```

Comment:
Don't need this either I think. (Need to re-run dist generation for this PR to pass)

```suggestion
```

Reply:
This one's not giving me an error so I think this should probably be kept?

Reply:
I see, yes, I guess we could keep it. Doesn't matter much at the moment, though, given it is only in Chromiums.